### PR TITLE
Update the OpenMP implementations of normal and FPFH estimation

### PR DESCRIPTION
**Pull request by ThorstenHarter**
The OpenMP implementations in the "features" module are missing bug fixes which have already been applied to the standard implementations some time ago.

This PR fixes two issues:
- NormalEstimationOMP should set the normal-cloud to non-dense when the computation fails
- FPFHEstimationOMP must skip invalid points or the kdtree-search will assert

### CODE REVIEW

**Comment by ThorstenHarter** [comment]
The gcc build of normal_3d_omp.hpp failed, although the code is the same as in normal_3d.hpp (besides the #pragma omp and type of idx, but I have that already ruled out).
Visual Studio 2015 build works fine.
I'll have to review this file again tomorrow...

**Comment by ThorstenHarter** [comment]
I have fixed the gcc/clang build and squashed my commits.
The AppVeyor build fails with a CMake error, I don't think that this is related to my changes.

File: features/include/pcl/features/impl/fpfh_omp.hpp [hunk 1]
```diff
@@ -62,7 +62,8 @@ pcl::FPFHEstimationOMP<PointInT, PointNT, PointOutT>::computeFeature (PointCloud
     for (size_t idx = 0; idx < indices_->size (); ++idx)
     {
       int p_idx = (*indices_)[idx];
-      if (this->searchForNeighbors (p_idx, search_parameter_, nn_indices, nn_dists) == 0)
+      if (!isFinite((*input_)[p_idx]) ||
```
Comment (taketwo) [inline features/include/pcl/features/impl/fpfh_omp.hpp]: Please add a space between function name and braces to match PCL style. Also in the hunk below.

Comment (ThorstenHarter) [reply features/include/pcl/features/impl/fpfh_omp.hpp]: Hi Sergey, I have added a space after each of the two calls to isFinite.

**Pull request merged**